src/components/Reports/SavingsFundTransfer/TransfersPage/TransfersPage.tsx

```diff
@@ -88,28 +93,42 @@ export const TransfersPage: React.FC<TransfersPageProps> = ({ title }) => {
       (reportData?.reportsSavingsFundTransfer ?? []).map((tx) => {
         return {
           ...tx,
-          transactedAt: DateTime.fromISO(tx.transactedAt),
+          transactedAt: DateTime.fromISO(tx.transactedAt, { setZone: true }),
           recurringTransfer: tx.recurringTransfer
             ? {
                 ...tx.recurringTransfer,
                 recurringStart: tx.recurringTransfer.recurringStart
-                  ? DateTime.fromISO(tx.recurringTransfer.recurringStart)
+                  ? DateTime.fromISO(tx.recurringTransfer.recurringStart, {
+                      setZone: true,
+                    })
                   : null,
                 recurringEnd: tx.recurringTransfer.recurringEnd
-                  ? DateTime.fromISO(tx.recurringTransfer.recurringEnd)
+                  ? DateTime.fromISO(tx.recurringTransfer.recurringEnd, {
+                      setZone: true,
+                    })
                   : null,
               }
             : null,
+          baseAmount: tx.amount || 0,
+          failedCount: 0,
+          summarizedTransfers: null,
+          missingMonths: null,
         };
       }),
     [reportData],
   );
 
-  const filteredTransactions = useFilteredTransfers(transactions);
+  const filteredTransactions = useMemo(
+    () => filteredTransfers(transactions),
+    [transactions],
+  );
 
-  const transferHistory: TransferHistory[] = filteredTransactions.map((tx) => {
-    const isRecurring = !!tx.recurringTransfer?.id;
+  const transferHistory: Transfers[] = filteredTransactions.map((tx) => {
+    const isRecurring = !!tx.recurringTransfer;
     const status = getStatusLabel(tx);
+    const shouldShowActions = () => {
+      return status !== StatusEnum.Pending && status !== StatusEnum.Ongoing;
+    };
 
     return {
```

Comment:
May of these default values aren't needed because many of the `tx` fields aren't nullable.